src/AzureAppConfigurationImpl.ts

```diff
@@ -29,20 +35,32 @@ export class AzureAppConfigurationImpl extends Map<string, unknown> implements A
         super();
         // Enable request tracing if not opt-out
         this.requestTracingEnabled = requestTracingEnabled();
-        if (this.requestTracingEnabled) {
-            this.enableRequestTracing();
-        }
 
         if (options?.trimKeyPrefixes) {
             this.sortedTrimKeyPrefixes = [...options.trimKeyPrefixes].sort((a, b) => b.localeCompare(a));
         }
+
+        if (options?.refreshOptions) {

```

Comment:
TODO: `watchedSettings` doesn't support wildcard, we should validate somewhere, and throw error if `*` detected.

Reply:
checked and added a test case. Now refreshOptions.watchedSettings will be validated and saved into a list `_sentinels`.